uint8 flag_velocity = 5
uint8 flag_angles = 2
uint8 flag_break = 13

uint8 flag
float64 data_x
float64 data_y
float64 data_z
float64 course
